homeassistant/components/elevenlabs/__init__.py

```diff
@@ -0,0 +1,21 @@
+"""The ElevenLabs text-to-speech integration."""
+
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
+from homeassistant.core import HomeAssistant
+
+PLATFORMS: list[Platform] = [Platform.TTS]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up ElevenLabs text-to-speech from a config entry."""
+    await hass.config_entries.async_forward_entry_setups(entry, PLATFORMS)
```

Comment:
validate the API key still works before forwarding entry setup

Reply:
Once validated, set the AsynClient as `entry.runtime_data` to be consumed in `tts.py`